### DISCUSSION TITLE
Sonic 3&K Sprites

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#144] [Q&A]
How do I replace Sonic's Sprites with the ones that are in the style of Sonic 3&K?

**Comment by DimensionWarped** [comment]
Do you know how to edit an image?

**Comment by DimensionWarped** [comment]
You know, I thought this sounded familiar...

https://github.com/Techokami/SonicWorldsNext/discussions/139

So you already asked the same question as this. You've already been given the answer. What gives?
